confignode/src/main/java/org/apache/iotdb/confignode/conf/SystemPropertiesUtils.java

```diff
@@ -124,11 +124,11 @@ public static void checkSystemProperties() throws IOException, ConfigurationExce
       needReWrite = true;
     } else if (!configNodeConsensusProtocolClass.equals(
         conf.getConfigNodeConsensusProtocolClass())) {
-      throw new ConfigurationException(
-          "config_node_consensus_protocol_class",
+      LOGGER.warn(
+          "[SystemProperties] The parameter \"config_node_consensus_protocol_class\" can't be modified after first startup. Your configuration: {} will be forced update to: {}",
```

Comment:
Please fix the code smell of these long one-character lines, and we can also extract these same strings to unify them

Reply:
Fixed～